packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -254,6 +259,9 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
         relayData.realizedLpFeePct = realizedLpFeePct;
         relayData.slowRelayer = msg.sender;
 
+        bytes memory ancillaryData = getRelayAncillaryData(depositData, relayData);
```

Comment:
By calling this, I think we're doing the hexToUtf8 call twice (once here and once in the event where we compute the relayHash).

I think this will reduce gas by ~50k:
```
bytes32 relayHash = _getRelayHash(depositData, relayData);
bytes memory ancillaryData = _getRelayAncillaryData(relayHash);
relayRequestAncillaryData[ancillaryData] = depositHash;
...
emit DepositRelayed(
    relayId,
    depositData,
    msg.sender,
    address(l1Token),
    realizedLpFeePct,
    depositHash,
    relayHash
);
```